src/main/java/step02/LottoCount.java

```diff
@@ -0,0 +1,19 @@
+package step02;
+
+import static step02.LottoConfig.LOTTO_PRICE;
+
+public class LottoCount {
+    private int count = 0;
```

Comment:
클래스 변수이므로 따로 0으로 초기화는 안하셔도 될 것 같습니다 😄 
그리고 `count`를 초기화 하는 생성자가 있으므로 `final`로 선언하셔도 될 것 같습니다 🙇 
```suggestion
    private final int count;
```